### ISSUE TITLE
Clarify language about public data

### ISSUE THREAD

**Issue by stvnrlly** [DCgov/economic-intelligence#37]
In particular:

> This site is an online version of the Deputy Mayor for Planning and Economic Development’s weekly dashboard. This dashboard is also transmitted to the City Administrator, the Mayor, and other senior staff, so they can be aware of economic trends and context. It includes only data that is public, so certain indicators that DMPED uses are not included.

Does this mean that this dashboard and the internal dashboard _both_ include only public information, or that this is a version of that internal dashboard that doesn't include non-public information?

**Comment by mseew** [comment]
Good point -- better to say "This version only includes data that is public..." to show that the online dashboards only contains visuals with datasets which align with our open licensing? The internal dashboard includes proprietary data.
